tests/hpc/ww4.pm

```diff
@@ -0,0 +1,150 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Configure and run a warewulf4 controller
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base qw(hpcbase hpc::utils), -signatures;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use lockapi;
+use utils;
+use Utils::Logging 'export_logs';
+use hpc::formatter;
+use isotovideo;
+use mm_tests;
+use Utils::Logging 'save_and_upload_log';
+use POSIX 'strftime';
+
+our $file = 'tmpresults.xml';
+
+sub run ($self) {
+    select_serial_terminal();
+    configure_static_network('10.0.2.1/24');
+
+    my $user_virtio_fixed = isotovideo::get_version() >= 35;
+    my $prompt = $user_virtio_fixed ? $testapi::username . '@' . get_required_var('HOSTNAME') . ':~> ' : undef;
+
+    ensure_ca_certificates_suse_installed();
+    my $rt = zypper_call("in warewulf4");
+    test_case('Installation', 'ww4', $rt);
+
+    $rt = systemctl 'enable --now warewulfd';
+    test_case('systemd', 'ww4', $rt);
```

Comment:
start warewulfd after you edited warewulf.conf like you do in line 40 and following